samtranslator/parser/parser.py

```diff
@@ -27,10 +27,10 @@ def _validate(self, sam_template, parameter_values):
                 [InvalidTemplateException("'Resources' section is required")])
 
         validation_errors = SamTemplateValidator.validate(sam_template)
```

Comment:
Why leave this line uncommented?

Reply:
The problem is so widespread that it's not feasible for us to fix this due to other priorities. Additionally, the original intent of the schema was not so much for validation, but for linters, IDE plugins etc. and we now have https://github.com/awslabs/cfn-python-lint for that.

Still need the call to `validate()` as it does additional validation, not just schema.